pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -1553,7 +1553,7 @@ default-package-overrides:
   - shake-language-c ==0.10.0
   - shakespeare ==2.0.11.1
   - shell-conduit ==4.5.2
-  - ShellCheck ==0.4.4
+  - shellcheck ==0.4.4
```

Comment:
We cannot choose arbitrary names where; this file must refer to the packages under the name they have on Hackage.